src/templates/next/components/shared/Card.tsx

```diff
@@ -0,0 +1,109 @@
+import { SingleCardProps } from "~/common/InfoCards"
+import Button from "../Button"
+import { Heading } from "../../typography/Heading"
+
+interface KeyStatisticsProps extends SingleCardProps {
+  variant?: "horizontal" | "vertical"
+  className?: string
+}
+
+const ImageComponent = ({
+  src,
+  alt,
+  className,
+}: {
+  src: SingleCardProps["imageUrl"]
+  alt: SingleCardProps["imageAlt"]
+  className?: string
+}) => {
+  return (
+    <div className={`aspect-h-1 aspect-w-1 overflow-hidden ${className}`}>
+      <img
+        src={src}
+        alt={alt}
+        className="h-full w-full object-cover object-center"
+      />
+    </div>
+  )
+}
+
+const TextComponent = ({
+  title,
+  text,
+  buttonLabel,
+  buttonUrl,
+  className,
+}: {
+  text: SingleCardProps["text"]
+  title: SingleCardProps["title"]
+  buttonLabel: SingleCardProps["buttonLabel"]
+  buttonUrl: SingleCardProps["buttonUrl"]
+  className?: string
+}) => {
+  return (
+    <div className={`p-7 flex flex-col flex-grow gap-3 h-fit ${className}`}>
+      {title && <h4 className={Heading[4]}>{title}</h4>}
```

Comment:
this should be `text-content-strong` too
![Screenshot 2024-03-21 at 10 27 41 PM](https://github.com/isomerpages/isomer-components/assets/77665182/19726e0d-d41b-4388-8412-559022aa9e80)

Reply:
https://github.com/isomerpages/isomer-components/pull/38/commits/7e78290ec0e3bb89f63595adf4b182f32ca8bbf7